bin/watch

```diff
@@ -95,35 +99,39 @@ function compilationComplete() {
         mochaOptions.push(argv.f);
     }
     mochap = spawn(argv.mocha, mochaOptions);
-    const source = mochap;
     mochap.on("close", (code) => {
-        if (source === mochap) {
```

Comment:
I think kill was not terminating immediately mocha and in some cases logs from previous mocha process were mixed with a newer mocha process. Be careful with these. It may not be an issue now but these checks were added for a reason.

Reply:
Okay, I will reintroduce them if necessary.

Reply:
I did replace this with

```
if (mochap) ...
```

and eliminated the source variable, which I thought was redundant.